cg/meta/workflow/nf_analysis.py

```diff
@@ -344,6 +363,16 @@ def create_nextflow_config(self, case_id: str, dry_run: bool = False) -> None:
                 file_path=self.get_nextflow_config_path(case_id=case_id),
             )
 
+    def create_gene_panel(self, case_id: str, dry_run: bool) -> None:
+        """Create and write an aggregated gene panel file exported from Scout."""
+        LOG.info("Creating gene panel file")
+        bed_lines: list[str] = self.get_gene_panel(case_id=case_id, dry_run=dry_run)
+        if dry_run:
+            bed_lines: str = "\n".join(bed_lines)
+            LOG.debug(f"{bed_lines}")
```

Comment:
unsure if we want debug or info logging here, as `--dry-run` already implies that you are in a debugging more. Feel free to disregard
```suggestion
            LOG.info(f"{bed_lines}")
```

Reply:
I don't think we want this in the Logs as it can be quite long and would clutter it. Also, this is only useful for debugging.
I will leave it as it is

Reply:
leave it, but the production logs won't get clogged because we don't run it in dry-run